sim/models/iris_realsense/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<model>
+  <name>3DR Iris</name>
+  <version>1.0</version>
+  <sdf version='1.4'>iris_depth_camera.sdf</sdf>
+
+  <author>
+   <name>Lorenz Meier and Thomas Gubler</name>
+   <email>lorenz@px4.io</email>
+  </author>
+
+  <description>
+    This is a model of the 3DR Iris Quadrotor. The original model has been created by
+    Thomas Gubler and is maintained by Lorenz Meier.
```

Comment:
mention that it is the regular Iris (created by Lorenz & Thomas) and you added the sensor from <name of that repo>